api/capi/src/nntrainer.cpp

```diff
@@ -110,10 +111,10 @@ int ml_nnmodel_destruct(ml_nnmodel_h model) {
 
   ML_NNTRAINER_CHECK_MODEL_VALIDATION(nnmodel, model);
 
-  nntrainer::NeuralNetwork *NN;
+  std::shared_ptr<nntrainer::NeuralNetwork> NN;
   NN = nnmodel->network;
   NN->finalize();
-  delete NN;
+  // delete [] NN;
```

Comment:
Please remove the whole line, not commenting out it.

Reply:
yes. fiexd. thanks.